src/DeviceStatusPage/DeviceInfoCard.tsx

```diff
@@ -90,11 +90,10 @@ const DeviceInfoCard = ({
       </p>
       <h6>Credentials inhibited</h6>
       <p>{device.hasCredentialsInhibited ? 'True' : 'False'}</p>
-      <div className="mt-auto">
+      <div className="mt-auto d-flex flex-column flex-md-row flex-wrap gap-2">
```

Comment:
If it's a flex column with gaps, we might as well use a `Stack`

Reply:
I removed the commit, it will be included in a future PR.
Regarding your suggestion, I'm using flex since it can wrap the row if necessary, and also because it lets me use the viewport's breakpoints to display either a row or a column of buttons: I cannot do that with a `Stack`.